admin_guide/managing_networking.adoc

```diff
@@ -592,18 +583,21 @@ ConfigMap for `EGRESS_DESTINATION` in the environment section:
     ...
 ----
 
-Note that the egress router will not automatically update when the
-ConfigMap changes; you will need to restart the pod to get updates.
+[NOTE]
+====
+The egress router does automatically update when the ConfigMap changes. Restart
```

Comment:
Oops, no, that should have been "The egress router does NOT automatically update when...". The "not" got lost in the rewording.